src/main/scripts/org/reaktivity/specification/nukleus/kafka/streams/merged/merged.fetch.server.sent.close.with.message/client.rpt

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright 2016-2020 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property clientAddress "nukleus://streams/kafka#0"
+
+connect await ROUTED_CLIENT
+        ${clientAddress}
+    option nukleus:window 16
+    option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${kafka:beginEx()
+                               .typeId(nukleus:id("kafka"))
+                               .merged()
+                                   .capabilities("FETCH_ONLY")
+                                   .topic("test")
+                                   .partition(0, 1)
+                                   .build()
+                               .build()}
+
+connected
+
+read nukleus:data.ext ${kafka:matchDataEx()
+                             .typeId(nukleus:id("kafka"))
+                             .merged()
+                                 .partition(0, 1)
+                                 .progress(0, 2)
+                                 .key("key1")
+                                 .header("header2", "value2")
+                                 .build()
+                             .build()}
+read "Hello, world #A1"
+
+read notify RECEIVED_MESSAGE
+read await CLOSE_MERGED
```

Comment:
For readability, this should be `write await` and moved down closer to `write close`.
Also, suggest renaming this barrier to `CLOSE_MERGED_INITIAL`.